src/browser/stacktrace.ts

```diff
@@ -0,0 +1,108 @@
+import { ShallowStackFrames, applyStackFrames, captureRawStackFrames } from "../utils/stacktrace";
```

Comment:
I didn't want to pull all of the `src/utils/stacktrace` because it is completely not related to the browser.

I can pull it to the `src` root though